File: src/targets/java/nethttp/fixtures/multipart-form-data-no-params.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("http://mockbin.com/har"))
    .header("Content-Type", "multipart/form-data")
    .method("POST", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());

File: src/targets/dart/http/fixtures/multipart-form-data-no-params.dart
import 'package:http/http.dart' as http;

void main() async {

  final headers = {
    'Content-Type': 'multipart/form-data',
  };

  final response = await http.post(
    Uri.parse('http://mockbin.com/har'),
    headers: headers,
  );

  print(response.statusCode);
  print(response.body);

}